src/components/Shared/Emoji/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+// https://medium.com/@seanmcp/%EF%B8%8F-how-to-use-emojis-in-react-d23bbf608bf7
+const Emoji = props => (
```

Comment:
At first I was curious as to if there was a lib on npm that already did this but my quick googling didn't lead to a solution using one being as clean as this. Great way to manage errors! (probably going to steal this component for Janis)